pallene/parser.lua

```diff
@@ -240,11 +241,11 @@ local grammar = re.compile([[
                            / NAME (ASSIGN / COMMA) %{LocalOrExportRequiredVariable} )*
                         |} !.
 
-    toplevelfunc    <- (P  export_or_local FUNCTION NAME^NameFunc
+    toplevelfunc    <- (P  export_or_local P FUNCTION NAME^NameFunc
```

Comment:
When we removed the `:` in the decls we didn't need to add a P after the `COLON`. Would it be possible to do the same here?

Reply:
The `P` after `:` in `decl` points exactly to the `:` token. With the modifier we need to capture the range for `KEYWORD WHITESPACE*`, if I am not wrong.